Mlem/Views/Tabs/Settings/Components/Views/AlternativeIconCell.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  AlternativeIconCell.swift
+//  Mlem
+//
+//  Created by tht7 on 28/06/2023.
+//
+
+import SwiftUI
+
+struct AlternativeIconCell: View {
+    let icon: AlternativeIcon
+    let setAppIcon: (_ id: String?) async -> Void
+
+    var body: some View {
+        Button {
+            Task(priority: .userInitiated) {
+                await setAppIcon(icon.id)
+            }
+        } label: {
+            HStack {
+                getImage()
+                VStack(alignment: .leading) {
+                    Text(icon.name)
+                    if let author = icon.author {
+                        Text(author)
+                            .font(.footnote)
+                            .foregroundColor(.secondary)
+                    }
+                }
+                Spacer()
+                if icon.selected {
+                    Image(systemName: "checkmark")
+                }
+            }
```

Comment:
I think we want `.accessibilityElement(children: .combine)` here.

Reply:
Ohhh thanks, I need to learn about accessibility eventually